### PR TITLE
Frontend Polish for Newfold Large Card and Welcome Step

### PR DESCRIPTION
**Pull request by abhijitb**
modified:   src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss
modified:   src/OnboardingSPA/components/CardHeader/stylesheet.scss
modified:   src/OnboardingSPA/components/Header/stylesheet.scss
modified:   src/OnboardingSPA/components/NewfoldLargeCard/stylesheet.scss
modified:   src/OnboardingSPA/pages/Steps/GetStarted/Welcome/stylesheet.scss

### CODE REVIEW

**Comment by arunshenoy99** [comment]
Screenshot 2022-09-07 at 11 16 59 AM
Screenshot 2022-09-07 at 11 20 43 AM
Screenshot 2022-09-07 at 11 21 14 AM
Screenshot 2022-09-07 at 11 22 50 AM
Screenshot 2022-09-07 at 11 25 00 AM

This looks a lot better than the initial welcome page, a few things I noticed
1. Newfold Large card still needs some more handling, a lot of screens violate "In general, the middle row of squares the top one should be mostly visible and the bottom one should be mostly not visible with a little bit poking out from behind the card at most major breakpoints between 390px wide to 3840px", attached a 1-2 screenshots of the same to get an idea.
2. On smaller screens (iPhone 4) the Next button seems to be going out of the screen and I can't scroll to get to it and the tab panels also overflow out of the card.
3. Not sure if this is an issue with chrome dev tools but in one of the SS you can see vertical parallel blue borders showing up on the tabs.

**Comment by abhijitb** [comment]
All fixes done for iphone4 screen size and the logo in the background ... I also sometimes see the extra border for the tabs and there is no CSS there which would show a border ... mostly looks like a developer tools issue

File: src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss [hunk 1]
```diff
@@ -1,12 +1,15 @@
 .nfd-nav-card-button {
 	color: var(--nfd-onboarding-white);
 	background-color: var(--wp-admin-theme-color-darker-10);
-	height: 8vh;
-	width: 50vw;
-	margin: 0vw auto;
+	padding: 16px;
+	width: 90%;
```
Comment (officiallygod) [inline src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss:5]: Hey, @abhijitb Maybe we can use 100%? Or we can align the Button in middle? 
I saw that the button is a bit left-aligned due to which it was a bit weird.

Comment (officiallygod) [reply src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss:5]: I used this extension to see the whole spacing https://chrome.google.com/webstore/detail/debug-css/igiofjnckcagmjgdoaakafngegecjnkj

**Review by officiallygod — CHANGES_REQUESTED** [review]

File: src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss (hunk 1 above)
Comment (abhijitb) [reply src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss:5]: Seems pretty well aligned for me :-( 

welcome-step-alignments

**Pull request merged**